client/signup/config/flows-pure.js

```diff
@@ -137,9 +137,16 @@ export function generateFlows( {
 			lastModified: '2019-06-05',
 		},
 
-		// We don't yet show the previews for the 'blog' segment
 		'onboarding-blog': {
-			steps: [ 'user', 'site-type', 'site-topic', 'site-title', 'domains', 'plans' ],
+			steps: [
+				'user',
+				'site-type',
+				'site-topic-with-preview',
+				'site-title-with-preview',
+				'site-style-with-preview',
+				'domains',
+				'plans',
+			],
```

Comment:
Could we not just delete this flow and use `onboarding`?  If so, we'd have to remove [this line](https://github.com/Automattic/wp-calypso/blob/6615611f796f87a678e8ab0088a960644e97a23d/client/signup/steps/site-type/index.jsx#L20).